homework_1/zad1/solution/game.go

```diff
@@ -0,0 +1,28 @@
+package solution
```

Comment:
"solution" is not really a good package name. I suppose it's how you do it in university exams, but we want to name packages by what they are logically doing (e.g. `fizzbuzz`). Naming the package `solution` is not descriptive in terms of what it's doing.